spec/models/order_spec.rb

```diff
@@ -12,4 +12,21 @@
     it {should belong_to :user}
   end
 
+  describe "instance methods" do
+    before(:each) do
+      @user = create(:user)
+      @merchant = create(:merchant)
+      @order_1 = create(:order, user: @user)
+      @item_1 = create(:item, user: @merchant, price: 4.50)
+      @item_2 = create(:item, user: @merchant, price: 2.33)
+      @oi_1 = create(:order_item, item: @item_1, order: @order_1, quantity: 3, price_per_item: @item_1.price)
+      @oi_2 = create(:order_item, item: @item_2, order: @order_1, quantity: 1, price_per_item: @item_2.price - 0.33)
```

Comment:
Tests look great!
Just one question: why subtract .33?

Reply:
I added that because sometimes the item's price doesn't match the order's price_per_item -- I didn't want them to always be equal.  I noticed that the test was passing when it shouldn't have been because I was using the price instead of price_per_item when calculating an order's total cost.

Reply:
I'll add a comment above that line to make that clearer!

Reply:
Thank you for the explanation!  That's a great catch, excellent testing.